zpng.zig

```diff
@@ -13,6 +13,7 @@ pub const Image = struct {
     width: u32,
     height: u32,
     pixels: [][4]u16,
+    gamma: f32,
```

Comment:
The precision of `f32` is pretty low so this will lose information - either use `f64` or just keep it as a `u32` and let the user convert it.

Also would be good to move this field above `pixels` to keep metadata together.